_data/icons.yml

```diff
@@ -0,0 +1,141 @@
+# Icons on the footer
+behance: # e.g. "sam"
+  pre: "https://www.behance.net/" 
+  pos:
+  icon:
```

Comment:
Do you think we can remove the ones that are empty?

Reply:
Yes.